modules/installation-configure-proxy.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+// * installing/installing_azure/installing-azure-customizations.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-aws.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-vsphere.adoc
+
+ifeval::["{context}" == "installing-bare-metal"]
+:bare-metal:
+endif::[]
+
+[id="installation-configure-proxy_{context}"]
+= Configuring the cluster-wide proxy during installation
+
+Production environments can deny direct access to the Internet and instead have
+an HTTP or HTTPS proxy available. You can configure a new {product-title}
+cluster to use a proxy by configuring the proxy settings in the
+`install-config.yaml` file.
+
+ifdef::bare-metal[]
+[NOTE]
+====
+For bare metal installations, node IP addresses should be assigned from the `install-config.yaml` file's `networking.machineCIDR` field. If they are not, then the node IP addresses must be manually added to the `install-config.yaml` file's `proxy.noProxy` field.
+====
+endif::bare-metal[]
+
+.Prerequisites
+
+* An existing `install-config.yaml` file.
+
+.Procedure
+
+. Edit your `install-config.yaml` file and add the proxy settings. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: my.domain.com
+proxy:
+  httpProxy: http://<username>:<pswd>@<ip>:<port> <1>
+  httpsProxy: https://<username>:<pswd>@<ip>:<port> <2>
```

Comment:
Due to https://bugzilla.redhat.com/show_bug.cgi?id=1750650, a `https` url scheme is not supported in 4.2. Therefore, the example should be:
`httpsProxy: http://<username>:<pswd>@<ip>:<port> <2>`. It's my understanding that a release note is or has been created for this caveat. You may want to reference the release note in this doc or make sure user's are aware of the limitation.

Reply:
Looks like the WIP release notes PR has mentioned this as a known issue, if you want to make sure this is sufficient: http://file.rdu.redhat.com/~ahardin/09232019/OCP-4-2-release-notes/release_notes/ocp-4-2-release-notes.html#ocp-4-2-known-issues

I'll update the example here.